cleaner/cleaner.go

```diff
@@ -0,0 +1,38 @@
+package cleaner
```

Comment:
Could you move it to `util/interrupt` and name the main function `RegisterCleaner` ?

Reply:
Moved.